### PR TITLE
Code review #2

### PR DESCRIPTION
**Pull request by agawda**
Changes after the first review.

### CODE REVIEW

File: scripts/generate_reports.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```
Comment (mhbratek) [inline scripts/generate_reports.sh:1]: Good Idea :bulb:

File: scripts/run_sonar.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (mhbratek) [inline scripts/run_sonar.sh:1]: Good Idea :bulb:

**Review by mhbratek — APPROVED** [review]
Sometimes you can add more info in commits messages.
Good Job :1st_place_medal:

**Review by arturAR — APPROVED** [review]
Do your sheeps fighting?

**Review by ghost — APPROVED** [review]

**Pull request merged**